packages/attribute-registry-consumer/src/model/converter.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  AttributeKindV1,
+  AttributeV1,
+  PersistentAttribute,
+  PersistentAttributeKind,
+  persistentAttributeKind,
+} from "pagopa-interop-models";
+import { match } from "ts-pattern";
+
+export const fromAttributeKind = (
+  input: AttributeKindV1
+): PersistentAttributeKind =>
+  match(input)
+    .with(AttributeKindV1.CERTIFIED, () => persistentAttributeKind.certified)
+    .with(AttributeKindV1.DECLARED, () => persistentAttributeKind.declared)
+    .with(AttributeKindV1.VERIFIED, () => persistentAttributeKind.verified)
+    .otherwise(() => {
+      throw new Error(`Invalid AttributeKind: ${JSON.stringify(input)}`); // Force exhaustive match
```

Comment:
Sadly `protobuf-ts` adds a `UNSPECIFIED$` field for each enum ("protobuf-ts requires all enums to have a 0 value"). So even without the `ts-pattern` issue, we should map that value.
We're currently mapping the `UNSPECIFIED$` value with `undefined` and we use `otherwise(() => undefined)` to resolve the `ts-pattern` issue but because the `UNSPECIFIED$` value should never be possible throwing an error and return the converted `T` value instead of `T | undefined` it's a better choice